general-exploration-landcover_io.ipynb

```diff
@@ -38,6 +38,9 @@
     "\n",
```

Comment:
<p>The paths are not correct (also in the text above). We should have</p><p> </p><pre class="ql-syntax" spellcheck="false">general-exploration-landcover_io
</pre><p> </p><p>instead of </p><p> </p><pre class="ql-syntax" spellcheck="false">general-exploration-landcover
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Environmental-DS-Book/general-exploration-landcover_io/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>Sorry this was my bad. I changed the name of the project folder to <code>general-exploration-landcover_io</code> without veryfing where else it's used. I'll sort this issue out as part of pending task for the editor. Thanks @annefou</p>

Reply:
<p>ok, fixed now see commit&nbsp;<a href="https://github.com/Environmental-DS-Book/general-exploration-landcover_io/pull/2/commits/9ab8466ce58d2ba18bd03b253a68cd84433e1000" target="_blank">9ab8466</a>. For reproducibility, I'd suggest to use <code>os.path.join</code> to use the existing <code>notebook_folder</code> variable. For instance, change <code>lcxr.to_netcdf("general-exploration-landcover/MT-study-area.nc")</code> to <code>lcxr.to_netcdf(os.path.join(notebook_folder,"MT-study-area.nc"))</code>. How does it sound @<a href="https://github.com/jamesdamillington" target="_blank">jamesdamillington</a>? Same for other cells using <code>notebook_folder</code>. </p><p> </p><p> </p>

Reply:
<p>Yes, I agree @acocac, <code>os.path.join</code> is better. I have edited in the code and in the markdown. </p><p> </p><p>I have also added <code>decode_coords="all"</code> to the <code>xarray.open_dataset</code> call to ensure CRS info is read back in correctly which it wasn't previously.     </p>

Reply:
<p>closing conversation, resolved</p>